bin/util/util.sh

```diff
@@ -62,7 +62,7 @@ getContractCode() {
 getFabContract() {
     CONTRACT_CODE=$(getContractCode $1 $2)
     SALT=$(seth --to-bytes32 $(seth --from-ascii $3))
-   # checks if a env variable with name of the fab is defined
+    # checks if a env variable with name of the fab is defined
     FAB_ADDR="${!3}"
     if [ -z "$FAB_ADDR" ]
```

Comment:
Currently the priority is the following:

1.  use FAB_ADDR from config file if defined
2. check if FAB is deployed on network if not deploy it.

Is this still good?